vocode/streaming/streaming_conversation.py

```diff
@@ -368,7 +367,7 @@ async def process(
                     message.text,
                     synthesis_result,
                     item.interruption_event,
-                    TEXT_TO_SPEECH_CHUNK_SIZE_SECONDS,
```

Comment:
We'll still have this (now unused?) CONSTANT floating around... not a big deal, probably fine as is.

Reply:
good catch